### PR TITLE
Updated to have the latest versions

### PR DESCRIPTION
**Pull request by TitanTheDumDum**

### CODE REVIEW

File: Website/index.html [hunk 1]
```diff
@@ -3,14 +3,16 @@
 <link rel="preconnect" href="https://fonts.googleapis.com">
 <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
 <link href="https://fonts.googleapis.com/css2?family=Chivo+Mono:wght@300&display=swap" rel="stylesheet">
+<script src="https://www.gstatic.com/firebasejs/8.10.0/firebase-app.js"></script>
+<script src="https://www.gstatic.com/firebasejs/8.10.0/firebase-firestore.js"></script>
 <html>
     <head>
         <title>Triangulum</title>
-        <link rel="icon" type="image/x-icon" href="https://firebasestorage.googleapis.com/v0/b/triangulum-32bdb.appspot.com/o/triangulumIcon.png?alt=media&token=996dd237-3cf2-4e87-b2cf-e3a0668a8a04">
```
Comment (LemionLaLemon) [inline Website/index.html:9]: Please add 
<link rel="icon" type="image/x-icon" href="https://firebasestorage.googleapis.com/v0/b/triangulum-32bdb.appspot.com/o/triangulumIcon.png?alt=media&token=996dd237-3cf2-4e87-b2cf-e3a0668a8a04"> 
to line 9 (Site icon, required)

File: Website/styles.css [hunk 2]
```diff
@@ -1,3 +1,6 @@
+:root{
+    overflow: hidden;
+}
```
Comment (LemionLaLemon) [inline Website/styles.css:3]: Nice fix for the right scrolling issue! Great job!

File: Website/styles.css [hunk 3]
```diff
@@ -93,7 +96,6 @@ h1 {
 #title{
     width: 100%;
     height: 20vh;
-    margin-bottom: 0px;
```
Comment (LemionLaLemon) [inline Website/styles.css:96]: Please add back margin-bottom

File: Website/styles.css [hunk 4]
```diff
@@ -106,83 +108,43 @@ label{
...
-    background-color: yellow;
-  }
-
-  .slider.round {
-    border-radius: 34px;
-  }
-
-  .slider.round:before {
-    border-radius: 50%;
+#Ct {
```
Comment (LemionLaLemon) [inline Website/styles.css:135]: Next time, please document your IDs better. #Ct is hard to understand.

**Review by LemionLaLemon — CHANGES_REQUESTED** [review]
Please check all feedbacks submitted and fix it! Thank you!

File: Website/index.html (hunk 1 above)
Comment (TitanTheDumDum) [reply Website/index.html:9]: Just do it later :/

File: Website/styles.css (hunk 2 above)
Comment (TitanTheDumDum) [reply Website/styles.css:3]: That's just feedback, resolved.

File: Website/styles.css (hunk 3 above)
Comment (TitanTheDumDum) [reply Website/styles.css:96]: The margin-bottom was breaking the background and hr. It was removed for a reason.

File: Website/styles.css (hunk 4 above)
Comment (TitanTheDumDum) [reply Website/styles.css:135]: Not an issue, resolved

**Comment by LemionLaLemon** [comment]
Reasonable responses. Approved

**Pull request merged**